influxql/ast_test.go

```diff
@@ -258,7 +258,7 @@ func TestSelectStatement_HasWildcard(t *testing.T) {
 
 		// No GROUP BY wildcards, time only
 		{
-			stmt:     `SELECT mean(value) FROM cpu GROUP BY time(5ms)`,
+			stmt:     `SELECT mean(value) FROM cpu where time < now() GROUP BY time(5ms)`,
```

Comment:
wait, how does this test pass? This should throw an error because you're asking for 5ms blocks from the beginning of time to now.